src/keymap.h

```diff
@@ -141,6 +141,7 @@ enum xkb_action_type {
 };
 
 enum xkb_action_flags {
+    ACTION_FLAG_NONE = 0,
```

Comment:
To me, `0` for flag enums seems clearer than `NONE`. What is the tidy rule which wants this?

Reply:
> What is the tidy rule which wants this?

It’s:

> warning: switching on non-enum value without default case may not cover all cases [bugprone-switch-missing-default-case]

Let’s add a default case and call it a day.